CMakeLists.txt

```diff
@@ -1245,16 +1245,10 @@ if(VRPN_BUILD_SERVER_LIBRARY)
 	if(VRPN_INSTALL)
 		install(TARGETS
 			vrpnserver
-			ARCHIVE
-			DESTINATION
-			lib
-			COMPONENT
-			serversdk
-			PUBLIC_HEADER
-			DESTINATION
-			include
-			COMPONENT
-			serversdk)
+			RUNTIME DESTINATION bin COMPONENT serversdk
+			LIBRARY DESTINATION lib${LIB_SUFFIX} COMPONENT serversdk
+			ARCHIVE DESTINATION lib${LIB_SUFFIX} COMPONENT serversdk
```

Comment:
Sure, I totally understand the reasoning, which is why I didn't want to make this a sticking point - the patch is good, and certainly using LIB_SUFFIX is better than not :D.

The handy thing about gnuinstalldirs, if you're cool with using a new enough CMake version that has it, is that it's a standard module that distros will typically patch if they are doing anything weird with regards to where to place their libraries that hasn't made it upstream yet (went to go show you an example from debian and apparently it's been upstreamed).

One of my "newer projects" you can see this used in is here: https://github.com/OSVR/libfunctionality/blob/master/CMakeLists.txt note that this mixes GNUInstallDirs with putting DLLs and .exes in the same directory to avoid headaches on Windows, so it's not a super clear, pure example.